client/src/components/hooks/useStakeDeploys.js

```diff
@@ -78,7 +78,7 @@ export const useStakeFromValidators = (publicKey) => {
 		(async () => {
 			if (!publicKey) return;
 			const { data } = await dispatch(getTransferDeploysStatus(pendingStakes.map((stake) => stake.deployHash)));
-			if (data && data.some((item) => 'pending' !== item.status)) {
+			if (data && data.some((item) => 'pending' !== item.status && item.status !== 'undelegating')) {
```

Comment:
Minor: just for better readability, this should be:
`data?.some((item) => item.status !== 'pending'  && item.status !== 'undelegating')`